core/dopostrequestservice.class.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * @since 1.3.0 N°6012
+ */
+class DoPostRequestService
+{
+	public function DoPostRequest($sUrl, $aData, $sOptionnalHeaders = null, &$aResponseHeaders = null, $aCurlOptions = []){
```

Comment:
I don't get the point of this method, doesn't seem overloaded.

Reply:
this class is used to be able to mock Utils::DoPostRequest and test further parts of the collector that use this static method.

Reply:
cf :

static public function DoPostRequest($sUrl, $aData, $sOptionnalHeaders = null, &$aResponseHeaders = null, $aCurlOptions = [])
	{
		if (self::$oMockedDoPostRequestService) {
			return self::$oMockedDoPostRequestService->DoPostRequest($sUrl, $aData, $sOptionnalHeaders, $aResponseHeaders, $aCurlOptions);
		}